hack/run-e2e-conformance-virtual-cluster.sh

```diff
@@ -386,9 +407,17 @@ if [ -z $SKIP_TEST ]; then
     export JUNIT_OUTPUT="${root}/${TEST_REPORT_PATH}/conformance-test-report"
   fi
 
+  # Disable exit on error temporarily to gather cluster information
```

Comment:
should you remove this one?

Reply:
It's intentional: with `set -e`, if `run-e2e-conformance.sh` fails the script stops and no cluster info is gathered. I could use a trap, but it has been already used in L48 and trapping multiple times would create a mess in the script (see [SO](https://stackoverflow.com/questions/3338030/multiple-bash-traps-for-the-same-signal)).

Proposals:
a. I can move these changes to a different PR, as they are a little out of scope.
b. I can move the `cluster-info dump` command into `run-e2e-conformance.sh`, binding it to a trap

Reply:
sounds good we can leave it here